ansible/playbook.yaml

```diff
@@ -0,0 +1,69 @@
+- name: Update
+  hosts: managednodes
+  remote_user: ec2-user
+
+  vars_prompt:
+
+    - name: trello_api_key
+      prompt: What is your Trello API key?
+      private: false
+
+    - name: trello_api_token
+      prompt: What is your Trello API token?
+      private: false
```

Comment:
This is a very reasonable call right now with the VMs set up as they are, but I'd still be inclined to switch on `private: true` (for the token at least, technically the key isn't a secret) in order to code defensively and reduce the chance we get bitten by this in future - it's easy to imagine the playbook could get ported to run in a pipeline or on some other environment that means we definitely don't want the token being printed out in logs